docs/resources/local.md

```diff
@@ -50,3 +50,5 @@ Artifactory.\nThis may not be safe and therefore requires strict content moderat
 uploading content that may compromise security (e.g., cross-site scripting attacks).
 * `download_direct` - (Optional) When set, download requests to this repository will redirect the client to download
 the artifact directly from the cloud storage provider. Available in Enterprise+ and Edge licenses only.
+* `cdn_direct` - (Optional) When set, download requests to this repository will redirect the client to download
```

Comment:
Let's use the tooltip text from Artifactory UI

<img width="495" alt="Screenshot" src="https://user-images.githubusercontent.com/27263/218870110-a64392d6-eb3a-4695-a11b-8b37af24b994.png">